ts-typeorm/test/pulumi/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as aws from "@pulumi/aws";
+import { PolicyPack, validateResourceOfType, StackValidationPolicy } from "@pulumi/policy";
+
+const stackPolicy: StackValidationPolicy = {
+    name: "eks-test",
```

Comment:
same copy paste error

Reply:
also missing lambda check potentially

Reply:
yeah these are missing a fair amount of the checks,  will have to iterate and get them in